bubble_sort.rb

```diff
@@ -0,0 +1,39 @@
+# rubocop: disable all
```

Comment:
- [ ] **MANDATORY** Please, do not disable rubocop linters, they are the guidelines for good coding practices in Microverse. Please let them do their work and fix all offenses that appear. You have 4 offenses
![image](https://user-images.githubusercontent.com/53256139/94779950-50744000-038d-11eb-9dfc-06992f96a0a5.png)